README.md

```diff
@@ -2,27 +2,24 @@
 
-1. Code management. Collaborating around code. Distributed version control. Git. Github. Issue tracking. Code review and pull requests. Branches and merging. Software licenses. Citing software. Software sustainability.
-1. Testing scientific software. Unit testing, regression testing. Test driven design. Expectations and assertions. Mocking. Build-and-test servers. Negative testing. Sensible error messages. Managed logging. Debugging and debuggers. Coverage measurement.
+1. Code management. Collaborating around code. Distributed version control. Git. Github. Issue tracking. Code review and pull requests. Branches and merging. Continuous Integration. Citing software. Software sustainability.
+1. Testing scientific software. Unit testing, regression testing. Test driven design. Expectations and assertions. Mocking. Negative testing. Sensible error messages. Error handling. Debugging and debuggers.
 1. Best practice in construction. Comments. Coding conventions. Documentation. Refactoring. IDEs. Configuration files. Using libraries.
 1. Design and development. Object oriented design. Software as engineering. Pragmatic use of diagram languages. Requirements engineering. Agile and Waterfall. Functional and architectural design.
-1. Building and deployment. Build tools. CMake and Scons.
-   Engineering for portability. Packaging. Deploying.
+1. Building and deployment. Build tools. Packaging. Deploying. Open Source community and interaction.
 
 ## Prerequisites:
 
-You must have reasonable experience in at least one compiled language, such as C++, C, or Fortran, and at least one dynamic language, such as Python, Ruby, Matlab or R.
-You must also have experience of the Unix shell.
+Lesson Zero covers installation of Python and Jupyter Notebooks, and we recommend you go through this material _before_ starting the course.
 
-Examples and exercises for this course will be provided in C++ and Python.  You will therefore find it easiest to follow along if you have experience in at least one of these languages. Appropriate Python experience could be obtained from the Software Carpentry Bootcamp, while C++ experience can be obtained from the RITS C++ course. Previous experience with version control (such as from Software Carpentry) would be helpful.
+Examples and exercises for this course will be provided in Python.  You will therefore find it easiest to follow along if you have experience in that language. Appropriate Python experience could be obtained from the Software Carpentry Bootcamp. Previous experience with version control (such as from Software Carpentry) would be helpful.
```

Comment:
I've removed C++ from here, not that there's any C++ content as it currently stands. It would be less accessible if the course expected two (very different) programming languages. Let's tell people how to do software engineering, then still be here when they ask "but how do I do it in C++".